### PR TITLE
Fixup build & packaging

### PR DESCRIPTION
**Pull request by nicoonoclaste**
- [x] Add the `recv.ml` cohttp example.
- [x] Fixup ocamlbuild & opam dependencies
- [x] Make running the test possible

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@ tmp
 *~
 \.\#*
 \#*#
+*.depends
```
Comment (neubyi) [inline .gitignore:6]: what is .depends?

File: _tags [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 
-true: package(astring sexplib)
-
 <src/*>: package(lwt lwt.unix)
 <src/conf.ml>: package(ppx_sexp_conv)
 <src/recv.*>: package(cohttp cohttp-lwt-unix)
 
-<test/*>: package(alcotest)
+<test/*>: package(alcotest fmt)
+<test/test.native>: package(sexplib)
```
Comment (neubyi) [inline _tags:12]: can you explain why only .native needs sexplib?

**Comment by neubyi** [comment]
Thanks!

File: _tags (hunk 2 above)
Comment (nicoonoclaste) [reply _tags:12]: Oh, bytecode builds probably do too  :/

File: .gitignore (hunk 1 above)
Comment (nicoonoclaste) [reply .gitignore:6]: An extension usually used for files generated by `ocamldep` and similar tools

**Comment by nicoonoclaste** [comment]
You are welcome  :)

**Pull request merged**